automation_tools/__init__.py

```diff
@@ -1832,68 +1858,68 @@ def sync_capsule_tools_repos_to_upgrade(admin_password=None):
     run(initials+'capsule content synchronize --id {0}'.format(capsule_id))
```

Comment:
for future, create a wrapper for run, something like run_prefix() in which you can prefix `initials = 'hammer -u admin -p {0} '.format(admin_password)` and run it.  This will avoid using `initials+` everytime.